packages/fx-core/src/component/driver/aad/error/aadManifestError.ts

```diff
@@ -0,0 +1,37 @@
+import { getLocalizedString } from "../../../../common/localizeUtils";
```

Comment:
Add the copyright header like other files. There're also several other files in this PR needs the header. Please check and update. I'm not going to add this comment repeatedly.